app/src/main/res/layout/activity_main.xml

```diff
@@ -53,13 +53,27 @@
                 android:textColor="@color/dark_gray"
                 android:textSize="18dp"
                 android:background="@drawable/add_item_border"
-                android:layout_marginRight="55dp"
+                android:layout_marginRight="97dp"
```

Comment:
Thank you so much for the video! That is not the look I was going for; I suggested the wrong code, sorry! :(

We should keep the 55dp marginRight so the input doesn't overlap with the button but add a paddingRight of 44dp as well so the typed in text doesn't overlap with the microphone icon (the icon is still within the text input).

Reply:
The video of the update:
https://photos.app.goo.gl/wVeuWHSdMDKFrtNP8